### PR TITLE
upd(webapp-manager-deb): `1.1.5` -> `1.1.9`

### PR DESCRIPTION
**Pull request by cat-master21**

### CODE REVIEW

**Review by wizard-28 — COMMENTED** [review]
@Korkmatik do you wish to continue maintaining this pacscript?

**Comment by Korkmatik** [comment]
@wizard-28 No I do not want to maintain it anymore

**Comment by cat-master21** [comment]
Not sure what the Ci's issue is.

File: packages/webapp-manager-deb/webapp-manager-deb.pacscript [hunk 1]
```diff
@@ -1,7 +1,8 @@
 name="webapp-manager-deb"
 gives="webapp-manager"
+repology="webapp-manager"
```
Comment (wizard-28) [inline packages/webapp-manager-deb/webapp-manager-deb.pacscript:3]:
```suggestion
repology=("project: webapp-manager")
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Review by wizard-28 — APPROVED** [review]

**Comment by Zahrun** [comment]
@cat-master21 , I think the https address is not reachable. Should be replaced by http.

**Comment by cat-master21** [comment]
Your right, @Zahrun I'll fix this soon.

**Pull request merged**